social_feed/app/views.py

```
@@ -0,0 +1,19 @@
+from django.shortcuts import render
+from .models import Post
+
+def home(request):
+    posts = Post.objects.all()  # Fetch all posts (you can add filtering for friends)
+    return render(request, 'index.html', {'posts': posts})
+
+from django.http import JsonResponse
+from .models import Post
```

Ruff (0.8.2)

Redefinition of unused `Post` from line 2

Remove definition: `Post`

(F811)